listenbrainz/webserver/views/test/test_profile.py

```diff
@@ -94,6 +94,15 @@ def test_delete(self, mock_publish_data_to_queue):
         user = db_user.get(self.user['id'])
         self.assertIsNone(user)
 
+    
+    def test_delete_listens(self):
+        self.temporary_login(self.user['login_id'])
+        r = self.client.get(url_for('profile.delete_listens'))
+        self.assert200(r)
+
+        r = self.client.post(url_for('profile.delete_listens'), data={'token': self.user['auth_token']})
+        self.assertRedirects(r, url_for('user.profile', user_name=self.user['musicbrainz_id']))
```

Comment:
So we should also test that the listens have actually gotten deleted. 

Let's add an integration test which does the following:
1. submits a bunch of listens
2. checks that the listens have been written
2. makes a post request to this endpoint
3. checks that the listens have been deleted.

Reply:
The integration tests have been added.